docs/reference_guide.md

```diff
@@ -1847,6 +1849,42 @@ Examples in situ:
 [search /examples](https://github.com/iovisor/bcc/search?q=attach_xdp+path%3Aexamples+language%3Apython&type=Code),
 [search /tools](https://github.com/iovisor/bcc/search?q=attach_xdp+path%3Atools+language%3Apython&type=Code)
 
+### 10. attach_func()
+
+Syntax: ```BPF.attach_func(fn, attach_type [, attachable_fd [, flags)```
+
+Attaches a BPF function of the specified type.
```

Comment:
Attaches a BPF function of the specified type to a particular attachable_fd. if the attach_type is BPF_FLOW_DISSECTOR, the function is expected to attach to `current` net namespace and `attachable_fd` must be 0.

Reply:
thanks, adjusted.